.meta.toml

```diff
@@ -3,3 +3,6 @@
 [meta]
 template = "default"
 commit-id = "13d8d6c0"
+
+[codespell]
+additional-ignores = "ist,discreet"
```

Comment:
`discreet` is beginning to sound like a term that we could have in a meta list that we always ignore...  Or check if there is a way to tell codespell to accept both British and American English, which I guess is what the problem is with this word.  I don't know on which side of the pond which word is used. :-)

Reply:
Ah, so both words are correct and they mean something different.  Or shall we say: something discrete? ;-)

Then this seems in error in `codespell`: it complains that `discreet` is badly spelled, when in fact it is a fine word.  That is why currently we need to ask codespell to ignore it.

`discreet` is a css class that is widely used in Plone.  It matches the meaning: unobtrusive.

Reply:
I see `discreet -> discrete` was [added in the "rare" dictionary](https://github.com/codespell-project/codespell/pull/1074/files).

From `codespell --help`:

```
  --builtin BUILTIN-LIST
                        comma-separated list of builtin dictionaries to
                        include (when "-D -" or no "-D" is passed). Current
                        options are:
                        - 'clear' for unambiguous errors
                        - 'rare' for rare (but valid) words that are likely to
                          be errors
                        - 'informal' for making informal words more formal
                        - 'usage' for replacing phrasing with recommended
                          terms
                        - 'code' for words from code and/or mathematics that
                          are likely to be typos in other contexts (such as
                          uint)
                        - 'names' for valid proper names that might be typos
                        - 'en-GB_to_en-US' for corrections from en-GB to en-US
                        The default is 'clear,rare'.
```

So we may want to call `codespell --builtin clear`.  In the pre-commit config file that is:

```
args: [--builtin, clear]
```

So we only check the really clear errors.
@stevepiercy Does that sound good to you?

Reply:
Yes, but let's be discreet about it. Unless there is a way to set a custom dictionary item to override `rare`?

Maybe this, from [Usage](https://github.com/codespell-project/codespell#usage):

```
codespell -I path/to/custom-good-spellings.txt
```

which would contain:

```
discreet
```

Reply:
I created a discrete issue for this:
https://github.com/plone/meta/issues/57

In the current PR let's just accept what we have.  It will get updated if we change the central config in `plone/meta` and apply it.